web-frame/src/app/app.module.ts

```diff
@@ -9,13 +9,22 @@
  */
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-import { RouterModule } from '@angular/router';
+import { ExtraOptions, RouterModule } from '@angular/router';
+import { ConfigModule } from '~app-shared/config';
 import { CoreModule } from '~app-shared/core';
 import { SecurityModule } from '~app-shared/security';
 
 import { APP_ROUTES } from './app-routes';
 import { AppComponent } from './app.component';
 
+const ROUTER_CONFIG: ExtraOptions = {
+  initialNavigation: 'disabled',
+  onSameUrlNavigation: 'reload',
+  paramsInheritanceStrategy: 'always',
+  relativeLinkResolution: 'corrected',
+  urlUpdateStrategy: 'eager',
```

Comment:
Besides disabling the initial navigation, I've made some extra changes to the router config.

> https://angular.io/api/router/ExtraOptions